keyboards/sofle/keymaps/bastien-baloup_fr/readme.md

```diff
@@ -0,0 +1,5 @@
+![Bastien-Baloup_FR keymap for sofle keyboard](https://i.imgur.com/Zlv1dj1.jpg)
```

Comment:
```suggestion
![Bastien-Baloup_FR keymap for sofle keyboard](https://i.imgur.com/Zlv1dj1h.jpg)
```

Reply:
I'm curious, what's the difference between them ?

Reply:
The `h`. ;)

Specifically, adding the H uses a smaller, but still high res image, that is much more friendly to github's markdown rendering.  

When sharing the image on Imgur, it will actually list a bunch of these.

Reply:
ok, thanks for the info.
I'll try to remember it next time I use imgur for a readme. :)